libraries/helpers.rb

```diff
@@ -8,11 +8,8 @@ def api_fqdn_available?
     end
 
     def api_fqdn_resolves?
-      require 'resolv'
-      Resolv.getaddress(node['chef-server']['api_fqdn'])
-      return true
-    rescue
-      false
+      extend ChefIngredientCookbook::Helpers
+      fqdn_resolves?(node['chef-server']['api_fqdn'])
```

Comment:
See [this comment](https://github.com/chef-cookbooks/chef-ingredient/pull/36/files#r38191000) about using `module_function` to make these sorts of method calls less awkward.